shell/common/api/api.mojom

```diff
@@ -64,11 +70,18 @@ interface ElectronBrowser {
     bool send_to_all,
     int32 web_contents_id,
     string channel,
-    mojo_base.mojom.ListValue arguments);
+    blink.mojom.CloneableMessage arguments);
 
   MessageHost(
     string channel,
-    mojo_base.mojom.ListValue arguments);
+    blink.mojom.CloneableMessage arguments);
+
+  // This is an API specific to the "remote" module, and will ultimately be
```

Comment:
#if BUILDFLAG(ENABLE_REMOTE_MODULE)

Reply:
Mojom files can't have `#if` in them unfortunately. I'm not sure it's worth splitting this out into an optionally-offered interface, but I can make the implementations `NOTREACHED()`.

Reply:
https://cs.chromium.org/chromium/src/mojo/public/tools/bindings/mojom.gni?l=274 can help build optional features into mojom files.